.github/workflows/qc.yml

```diff
@@ -0,0 +1,31 @@
+# Basic ODK workflow
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  sssom_validation:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.4
+    steps:
+      - name: Install latest SSSOM
+        env:
+          DEFAULT_BRANCH: master
+        run: pip install --upgrade pip && pip install -U sssom 
```

Comment:
should this not install SSSOM locally from the repository that's been pulled, so it can be used to test changes to the code as well? Otherwise, this PR is great.

Reply:
This requires more work, because the sssom toolkit is in a different location, and maintained differently then the stuff maintained here. We know what to do: https://github.com/mapping-commons/sssom/issues/115

But we have not gotten around to this yet.